src/modules/universe/RadixNodeDiscoveryHardcodedSecure.ts

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios'
+
+import RadixNodeDiscovery from './RadixNodeDiscovery'
+
+import { RadixSerializer } from '../atommodel'
+import { RadixNode, RadixNodeSystem } from '../..'
+
+/**
+ * Radix node discovery from a fixed list
+ */
+export default class RadixNodeDiscoveryHardcodedSecure implements RadixNodeDiscovery {
+    
+    /**
+     * Creates an instance of radix node discovery from seed.
+     * @param bootstrapNode Full address to the rpc endpoint of a Radix node in the universe
+     */
+    constructor(
+        readonly bootstrapNodes: string[],
+        ) {}
+
+    public async loadNodes() {
+        return Promise.all(this.bootstrapNodes.map(async (address) => {
+            try {
+                const response = await axios.get(`https://${address}/api/system`)
```

Comment:
I may be mixing things up now, but I'll ask anyway (just for my own peace of mind): Are we sure about this endpoint? The `BootstrapByTrustedNode` implementation in the Java library uses the `/api/universe` endpoint for example. This is also where I suspect I may compare apples to pears.

Reference: https://github.com/radixdlt/radixdlt-java/blob/release/1.0.0-beta/radixdlt-java/src/main/java/com/radixdlt/client/core/BootstrapByTrustedNode.java#L31

Reply:
Yes, this is a different mechanism — for node discovery.

The universe is hard coded in the JS library right now, so no universe discovery yet.